pkg/ee/resource-quota/resource-quota-synchronizer/synchronizer_test.go

```diff
@@ -152,6 +156,9 @@ func genResourceQuota(name string, deleted bool) *kubermaticv1.ResourceQuota {
 	rq := &kubermaticv1.ResourceQuota{}
 	rq.Name = name
 	rq.Namespace = kubermaticresources.KubermaticNamespace
+	rq.Labels = map[string]string{
+		kubermaticv1.ResourceQuotaSubjectNameLabelKey: "project1",
```

Comment:
nit: consider adding kind as well as we expect both labels to exist in regular objects.
```suggestion
		kubermaticv1.ResourceQuotaSubjectNameLabelKey: "project1",
		kubermaticv1.ResourceQuotaSubjectKindLabelKey: "project",		
		
```